docs/FAQs.md

```diff
@@ -0,0 +1,37 @@
+# Frequently Asked Questions
+
+## Should I Migrate from TSLint to ESLint?
+
+**Yes**.
+
+[TSLint is deprecated](https://medium.com/palantir/tslint-in-2019-1a144c2317a9) and will [only receive patches](https://github.com/palantir/tslint/issues/4534) for security vulnerabilities and breaking TypeScript changes.
+Even if it still works on your project, it will become less useful over time as TypeScript evolves.
+
+## Should I Use `tslint-to-eslint-config`?
+
+`tslint-to-eslint-config` is recommended for use if you require near-identical behavior in transitioning from TSLint to ESLint.
+This is most reasonable when your project is large enough that fixing for different linter rules would be a significant time investment.
+
+However, after -or even better, _before_- you're migrated to ESLint, we recommend you take this opportunity to re-evaluate your core lint rules.
+TSLint's recommendations were solidified several core TypeScript versions ago and don't always reflect the latest and greatest standards and lint rules.
+
+Our recommended TSLint-to-ESLint configuration migration approach is:
+
+1. Switch your configuration to extend from [typescript-eslint's `recommended` and `recommend-requiring-type-checking` rulesets](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md)
```

Comment:
maybe the first step in this ordered list should be actually _using_ `tslint-to-eslint-config`?

Reply:
Haha I was actually in the headspace of not doing that, since I've been just starting from scratch... but I can see how that would be a confusing recommendation. Plus a lot of the folks I've talked to consider starting from scratch a much more time-intensive, collaborative process than switching minimally... will edit!